covdefaults.py

```diff
@@ -58,22 +59,35 @@ def _plat_impl_pragmas():  # type: () -> List[str]
 
 
 class CovDefaults(CoveragePlugin):
-    def __init__(self, subtract_omit: str = '') -> None:
+    def __init__(
+        self,
+        subtract_omit: str = '',
+        installed_package: str = '',
+    ) -> None:
         self._subtract_omit = subtract_omit.split()
+        self._installed_package = self._parse_inst_pkg(installed_package)
+
+    @staticmethod
+    def _parse_inst_pkg(installed_package: str) -> Dict[str, Optional[str]]:
+        result: Dict[str, Optional[str]] = {}
+        for entry in installed_package.split(' '):
+            if entry:
+                key, value = entry, None
+                if entry.count(':') == 1:
+                    key, value = entry.split(':')
+                result[key] = value
+        return result
```

Comment:
this mapping seems weird, sometimes it's `{'pkgname': None}` other times it's `{'virtualenv': '.'}`?

Reply:
In theory, we could consider None to default to `.` but I've thought to make that decision later.